project-minimization-plugin/src/test/resources/logback.xml

```diff
@@ -0,0 +1,68 @@
+<configuration>
+
+    <!-- appender for console output -->
+    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <!-- appender for TRACE and DEBUG -->
+    <appender name="DEBUG_FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>logs/logs.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+            <fileNamePattern>logs/logs-%d{yyyy-MM-dd}.log</fileNamePattern>
+            <maxHistory>30</maxHistory>
+        </rollingPolicy>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <!-- Appender for STATISTICS -->
+    <appender name="STATISTICS_FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>logs/statistics.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+            <fileNamePattern>logs/statistics-%d{yyyy-MM-dd}.log</fileNamePattern>
+            <maxHistory>30</maxHistory>
+        </rollingPolicy>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <!-- appender for WORKING logs -->
+    <appender name="WORKING_FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>logs/working.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+            <fileNamePattern>logs/working-%d{yyyy-MM-dd}.log</fileNamePattern>
+            <maxHistory>30</maxHistory>
+        </rollingPolicy>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %class{36} - %msg%n</pattern>
+        </encoder>
+
+    </appender>
+
+    <!-- Logger for STATISTICS (write only in statistics.log) -->
+    <logger name="STATISTICS" level="INFO" additivity="false">
+        <appender-ref ref="STATISTICS_FILE" />
+    </logger>
+
+    <!-- Logger for Working (write only in working.log) -->
+    <logger name="WORKING" level="INFO" additivity="false">
+        <appender-ref ref="WORKING_FILE" />
+    </logger>
+
+    <!-- Logger for TRACE and DEBUG (write only in logs.log) -->
+    <logger name="TRACE_DEBUG" level="TRACE" additivity="false">
+        <appender-ref ref="DEBUG_FILE" />
+    </logger>
```

Comment:
It's not necessary for tests

Reply:
I think trace/debug are pretty usefull for testing(?)

Reply:
Yes, but when we want to debug, we will turn it on manually